templates/SilverStripe/Forms/GridField/GridFieldSudoMode.ss

```diff
@@ -0,0 +1,5 @@
+<tr class="grid-field__title-row">
```

Comment:
This isn't the title row, don't use the title row's css class.

Reply:
Updated